index.html

```diff
@@ -888,15 +901,37 @@ <h2>Algorithm</h2>
 				<li>Otherwise, the result of the <a href="https://www.w3.org/TR/did-core/#method-operations">Read</a> operation
 					is called the <var>output <a>DID document</a></var>. This result MUST be represented in a
 					<a href="https://www.w3.org/TR/did-core/#representations">conformant representation</a> that
-					corresponds to the <b>accept</b> <var>resolution option</var>.</li>
-				<li>Return the following result:
-					<ol class="algorithm">
-						<li><b>didResolutionMetadata</b>: <code>«[ ... ]»</code></li>
-						<li><b>didDocument</b>: <var>output DID document</var></li>
-						<li><b>didDocumentMetadata</b>: <code>«[ "contentType" → </code><var>output DID document media type</var><code>, ... ]»</code></li>
-					</ol>
-				</li>
+					corresponds to the <b>accept</b> <var>input <a href="#did-resolution-options">DID resolution option</a></var>.</li>
 			</ol>
+			<li>If the <var>input <a href="#did-resolution-options">DID resolution options</a></var> contain the <code>expandRelativeUrls</code> option with a value of <code>true</code>:
+				<ol class="algorithm">
+					<li>Iterate over all <a data-cite="did-core#services">services</a>,
+						<a data-cite="did-core#verification-methods">verification methods</a>, and
+						<a data-cite="did-core#verification-relationships">verification relationships</a>
+						in the <var>output <a>DID document</a></var>.</li>
+					<li>If the value of the <code>id</code> property of a <a data-cite="did-core#services">service</a> or
+						<a data-cite="did-core#verification-methods">verification methods</a> is a
+						<a data-cite="did-core#relative-did-urls">relative DID URL</a>, or if a
+						<a data-cite="did-core#verification-relationships">verification relationship</a> is a
```

Comment:
I think this needs slightly updating for the case when verification relationships are contain embedded verification methods.

Reply:
Hmm I think that case is covered by the current language here.. It says:

> If the value of the `id` property of a [..] verification method is a relative DID URL [..]

In my mind this covers all verification methods, both at the top level, and embedded in a verification relationship. But I'll try to clarify this a bit more.

Reply:
Hmm yes, I do think an implementer should be able to work out what to do.